pkgs/development/python-modules/pyqt-builder/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, fetchPypi, buildPythonPackage, packaging, sip_5 }:
+
+buildPythonPackage rec {
+  pname = "pyqt-builder";
+  version = "1.6.0";
+
+  propagatedBuildInputs = [ packaging sip_5 ];
+
+  src = fetchPypi {
+    pname = "PyQt-builder";
+    inherit version;
+    sha256 = "0g51yak53zzjs4gpq65i01cmpz7w8cjny9wfyxlgr2vi0wag107v";
+  };
+
+  meta = with lib; {
+    description = "PEP 517 compliant build system for PyQt";
+    homepage    = "https://pypi.org/project/PyQt-builder/";
```

Comment:
I would not indent like that but it is a detail.

Reply:
Unindent this. We don't need pretty-prinmting, seriously.

Reply:
@eduardosm

Reply:
What do you mean with "indent"?

Reply:
I'd assume they're talking about the alignment (where the equal signs are all lined up).

Reply:
I'll remove it, but I've seen many packages that way.